frontend/awx/access/credentials/components/CredentialMultilineInput.tsx

```diff
@@ -26,16 +32,24 @@ export function CredentialMultilineInput({
   accumulatedPluginValues: CredentialPluginsInputSource[];
   setAccumulatedPluginValues?: (values: CredentialPluginsInputSource[]) => void;
   setPluginsToDelete?: React.Dispatch<React.SetStateAction<string[]>>;
+  isSecret?: boolean;
+  pluginsToDelete?: string[];
+  initialValues?: initialValues;
 }) {
   const { t } = useTranslation();
+  const [shouldHideField, setShouldHideField] = useState(isSecret);
```

Comment:
Since we have access to `field` already, we can set isSecret to just field.secret instead of passing it as a prop.